dora/core/server/worker/src/main/java/alluxio/worker/dora/PagedDoraWorker.java

```diff
@@ -731,35 +731,29 @@ public OpenFileHandle createFile(String path, CreateFilePOptions options)
     if (options.hasRecursive() && options.getRecursive()) {
       createOption.setCreateParent(true);
     }
-
-    try {
```

Comment:
Why is the outer most `try` statement removed?

The original code is to catch any IOException here and convert it to a RuntimeException().
Because RuntimeException is expected on gRPC client side, IIRC.
Please double check the necessity of this change.